src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -734,8 +745,17 @@ struct ScaledDotProductAttention::AttentionExecutor : public ScaledDotProductAtt
             //  1, in matrix mutiply, using AMX is not efficency because the M dimension of A will alway be 1
             //  2, using float will save the repack cost which typically is required for bf16/int8 opt
             //  3, using dot product can leverage the SIMD while easily adapt to indirect kv cache
-            kernel_single_token(m_query_emb, present_key, present_value, {}, use_attn_mask ? attn_mask : PlainTensor<float>(),
-                        output_emb, beam_table, has_out_transpose, auto_causal, scale_input);
+            if (fuse_concat) {
+                PlainTensor<T2> present_key = k_cache, present_value = v_cache;
+                MHASingleToken<T, T2> kernel_single_token;
```

Comment:
Better to keep `kernel_single_token` as the class member which has some cached reusable buffer